mllib/src/main/scala/org/apache/spark/ml/clustering/GaussianMixture.scala

```diff
@@ -104,6 +105,17 @@ class GaussianMixtureModel private[ml] (
   @Since("2.0.0")
   def gaussians: Array[MultivariateGaussian] = parentModel.gaussians
 
+  /**
+   * Helper method used in Python
+   */
+  def gaussiansDF: DataFrame = {
+    val tmp_gaussians = parentModel.gaussians
+    val modelGaussians = tmp_gaussians.map { gaussian =>
+      (gaussian.mu, gaussian.sigma)
+    }
+    sqlContext.createDataFrame(modelGaussians).toDF("mu", "sigma")
```

Comment:
Let's call the columns "mean" and "cov"
I'm planning on naming them mean/cov in the spark.ml MultivariateGaussian in order to match other libraries.